### ISSUE TITLE
Create first version of the landing page

### ISSUE THREAD

**Issue by pjeziorowski** [appwish/landingpage#1]
We need to have a landing page. There is a mockup done already:
- https://xd.adobe.com/view/2004c923-3980-49d0-6769-e6fcc730a42a-5635/

In this task, you have to create a landing page that matches the mockup from above.
You can use any technique / tool / framework you wish to achieve this. You have a freedom to choose.

I'll soon add all required assets to this description e.g. all SVG files you need.

**Comment by Hina-softwareEngineer** [comment]
@pjeziorowski Can I Make it in html5 and Css? Or you need other framework or tool?

**Comment by pjeziorowski** [comment]
@Hina-softwareEngineer HTML and CSS should be enough :)

**Comment by pjeziorowski** [comment]
Hello @Hina-softwareEngineer , do you work on this task or this can be taken by another person?

**Comment by Hina-softwareEngineer** [comment]
Yeah! I am working. 70% is completed. Give me two days more. So, I could complete and also make it responsive.

**Comment by pjeziorowski** [comment]
That's great @Hina-softwareEngineer <3
